common/nextpnr.cc

```diff
@@ -221,6 +221,9 @@ delay_t Context::getNetinfoRouteDelay(const NetInfo *net_info, const PortRef &us
         return 0;
 #endif
 
+    if (net_info->wires.empty())
```

Comment:
What's the impact of this change? Just a speedup?

Reply:
Yes, makes timing analysis significantly faster for unrouted designs (it would always call the bottom predictDelay in this case, just avoids the extra getNetinfoSourceWire etc calls)